view/frontend/web/js/view/express-pay.js

```diff
@@ -48,7 +48,17 @@ define([
                                 } else {
                                     messageList.addErrorMessage({ message: $t('An error occurred while processing your payment. Please try again.') });
                                 }
-                            }
+                            },
+                            async onShippingAddressChange(data, actions) {
+                                expressPay.updateQuoteShippingAddress(data['shippingAddress']);
+
+                                try {
+                                    await expressPay.updateOrder(data['orderID']);
+                                    return actions.resolve();
```

Comment:
Does calling resolve here change the behaviour at all? I never saw it in their documentation.

Reply:
Ooh yes you're right, looks like it's only the deprecated `onShippingChange` that has a resolve action. I've removed this one.